lib/honeycomb/span.rb

```diff
@@ -36,11 +36,12 @@ def initialize(trace:,
       parse_options(**options)
     end
 
-    def parse_options(parent: nil,
+    def parse_options(parent: nil, # rubocop:disable Metrics/ParameterLists
```

Comment:
I opted to disable rubocop on this line rather than jump through the hoops to refactor or group the parameters, mainly because it looks like we'll be shaving one param back off here in the next major version. Lemme know what you think though, I know it's cheating a little :)